pkg/apis/apis/v1alpha1/types.go

```diff
@@ -378,6 +384,14 @@ type APIExportPolicy struct {
 // of the API Export
 type LocalAPIExportPolicy struct{}
 
+type PermissionClaim struct {
+	metav1.GroupResource `json:","`
+
+	// This is the identity for a given API Resource, It will be empty for core types.
+	// Note that one must look this up for a particular KCP instance.
```

Comment:
worth to mention that it can be found in APIExport and APIResourceSchema status?

Reply:
Added a note